pkg/drivers/kic/oci/oci.go

```diff
@@ -554,6 +554,23 @@ func ListContainersByLabel(ctx context.Context, ociBin string, label string, war
 	return names, err
 }
 
+// ListImagesByContainerLabel returns all the images names with a specified container label
+func ListImagesByContainerLabel(ctx context.Context, ociBin string, label string) ([]string, error) {
+	rr, err := runCmd(exec.CommandContext(ctx, ociBin, "ps", "-a", "--filter", fmt.Sprintf("label=%s", label), "--format", "{{.Image}}"))
+	if err != nil {
+		return nil, err
+	}
+	s := bufio.NewScanner(bytes.NewReader(rr.Stdout.Bytes()))
+	var names []string
+	for s.Scan() {
+		n := strings.TrimSpace(s.Text())
+		if n != "" {
+			names = append(names, n)
+		}
+	}
```

Comment:
After the scan loop should check for errors with `s.Err()` to ensure there possibly wasn't more images but an error occurred.